primitives/src/assets.rs

```diff
@@ -323,10 +335,9 @@ impl FungibleLedgerError {
 			WithdrawConsequence::NoFunds => Self::NoFunds,
 			WithdrawConsequence::Overflow => Self::Overflow,
 			WithdrawConsequence::Underflow => Self::Underflow,
-			WithdrawConsequence::ReducedToZero(balance) => Self::ReducedToZero(balance),
 			WithdrawConsequence::UnknownAsset => Self::UnknownAsset,
 			WithdrawConsequence::WouldDie => Self::WouldDie,
-			WithdrawConsequence::Success => return Ok(()),
+			WithdrawConsequence::Success | WithdrawConsequence::ReducedToZero(_) => return Ok(()),
```

Comment:
`ReducedToZero` is actually a success case, because it happens when the account doesn't have to be kept alive.